example/main.go

```diff
@@ -78,26 +90,30 @@ func getAndUpdateItem(client *onepassword.Client, existingVaultID, existingItemI
 	if err != nil {
 		panic(err)
 	}
+	// [developer-docs.sdk.go.update-item]-end
 
 	for _, f := range updatedItem.Fields {
 		if f.Title == "Details" {
-			doSomethingSecret(f.Value)
+			fmt.Println(f.Value)
 		}
 	}
 }
 
 func resolveSecretReference(client *onepassword.Client, vaultID, itemID, fieldID string) {
 	// Retrieves a secret from 1Password.
 	// Takes a secret reference as input and returns the secret to which it points.
+	// [developer-docs.sdk.go.resolve-secret]-start
 	secret, err := client.Secrets.Resolve(context.Background(), fmt.Sprintf("op://%s/%s/%s", vaultID, itemID, fieldID))
 	if err != nil {
 		panic(err)
 	}
+	// [developer-docs.sdk.go.resolve-secret]-end
 
-	doSomethingSecret(secret)
+	fmt.Println(secret)
```

Comment:
Is there any reason not to include the print part of this in the Go example, like we do for JS & Python?

Reply:
I included it now 😄